pkg/cmd/errors/login.go

```diff
@@ -0,0 +1,49 @@
+package errors
```

Comment:
So... although I suggested the CLI errors framework to reside in the upper level package `pkg/cmd/errors` so that it could be used by all CLI commands, I'm not confident this one belongs here too, once this is specific to `login` which is under `oc` in `pkg/cmd/cli/cmd`. Could probably stay in `pkg/cmd/cli/cmd/errors`. I'm not 100% convinced though. @juanvallejo WDYT?